core/variant/variant_construct.cpp

```diff
@@ -84,11 +84,13 @@ void Variant::_register_variant_constructors() {
 	add_constructor<VariantConstructNoArgs<Vector2>>(sarray());
 	add_constructor<VariantConstructor<Vector2, Vector2>>(sarray("from"));
 	add_constructor<VariantConstructor<Vector2, Vector2i>>(sarray("from"));
+	add_constructor<VariantConstructor<Vector2, double>>(sarray("x_and_y"));
```

Comment:
These should be moved down to not break compatibility

Reply:
I thought so, on it!